src/OmniSharp.Abstractions/Options/OmniSharpOptions.cs

```diff
@@ -4,6 +4,8 @@ namespace OmniSharp.Options
 {
     public class OmniSharpOptions
     {
+        public CodeActionOptions CodeActions { get; set; } = new CodeActionOptions();
```

Comment:
that's a good observation. Let's change it to `public CodeActionOptions CodeActions { get; } = new CodeActionOptions();` I think this should still bind fine to the ASP.NET Configuration and be more elegant. The other property (`FormattingOptions`) should use the same approach, it would be cleaner. Currently it can be set through the constructor too, but that constructor is redundant - I don't believe we use it anywhere, only the default constructor is in use